### ISSUE TITLE
Failed npm command and stdout

### ISSUE THREAD

**Issue by vmiko** [jfrog/jfrog-cli#2089]
### Describe the bug

Hello,

Since the jf release 2.38.0, running a npm command through the jf cli doesn't print stdout properly.
Indeed, when a npm command failed (with an exit code 1 for example), only stderr is returned by the jf cli.

Important information can be sent by npm on stdout when a npm command goes wrong. Without stdout returned, this is very hard to debug a npm command.

This behavior is not visible with a jf cli version < 2.37.3.
The latest jf version (2.43.1) has this behavior.

Thank you in advance

### Current behavior

By following the reproduction steps with a jf cli >= 2.38.0 you should see the following output :

```shell
jf npm run issue
15:09:19 [🚨Error] couldn't run npm run: stderr
```

### Reproduction steps

To reproduce a failed npm command, try to add a `script` in a `package.json` that returns an exit code 1.
Before to exit 1, print messages on stdout and stderr and observe the behavior.

A `package.json` sample
```json
...
"scripts": {
    "issue": "echo 'stdout'; echo 'stderr' >&2; exit 1",
  },
...
```

Then run the following command : `jf npm run issue`

### Expected behavior

When a npm command goes wrong it could be appreciated to returned both stderr and stdout like before the 2.38.0 release.

```shell
jf npm run issue

> myproject@0.0.1 issue
> echo 'stdout'; echo 'stderr' >&2; exit 1

stdout
stderr
14:46:14 [🚨Error] exit status 1
```

### JFrog CLI version

2.43.1

### Operating system type and version

Ubuntu 22.04.2 LTS

**Comment by gregoryboue** [comment]
@jannylund @phyber @btashton 
Is it possible to do something about this ?

**Comment by martinssonj** [comment]
Also experience this issue.

**Comment by vmiko** [comment]
I think having finding the root cause and [I raised a feature request the 16 june](https://github.com/jfrog/build-info-go/issues/169) but still sadly not answered. Please jfrog do something for us and our users🙏

**Comment by sverdlov93** [comment]
Hi @vmiko, @gregoryboue @martinssonj, @mthidell, @nantes44-github, @sstephant, and @gdickinson-nn
Thanks for reporting this issue.
Our apologies for the inconvenience caused by this bug.
We believe that the issue was [fixed on JFrog CLI 2.46.0](https://github.com/jfrog/jfrog-cli/releases/tag/v2.46.0) so feel free to update the cli version to the latest.
Looking forward to your feedback on it.

**Comment by vmiko** [comment]
Hi @sverdlov93 

Thank you for the fix and for taking my issue into consideration.
It works like a charm, I appreciate.

I close this issue.
Have a nice day
